specification_templates/worse.yaml

```diff
@@ -0,0 +1,35 @@
+data:
+  regression_version: '2020_12_01.12'
+  covariate_version: '2020_11_30.03'
+  output_root: ''
+scenarios:
+  worse:
+    algorithm: 'normal' # normal, draw_level_mandate_reimposition
+    solver: 'RK45'
+    system: 'normal'
+#    system_params:
+#        forecast_version: 'reference'
+#        proportion_immune: 0.5
+#    population_partition: 'high_and_low_risk'
```

Comment:
Did these args go away?

Reply:
This is a test specification.  Versioning config files is always a little iffy.

Reply:
I was being lazy here.  Not providing the args makes the defaults get used, which was the behavior I wanted.